src/components/ArtworkItem.jsx

```diff
@@ -10,11 +10,11 @@ const ArtworkItem = ({ data }) => (
     <img
       className="w-full h-64 py-4 px-4"
       src={data?.images?.web ? `${data.images?.web?.url}` : Desing1}
-      alt="producto"
+      alt={data.title ? data.title : "Item"}
     />
     <div>
       <div className="font-extrabold vertical-ellipsis overflow-hidden text-center text-ellipsis block mx-6 text-2xl xl:text-xl 2xl:text-2xl my-4 leading-6">
-        {data.title ? data.title : "Nombre del producto"}
+        {data.title ? data.title : "Item"}
```

Comment:
What do you think about use:
```suggestion
      alt={data.title ? data.title : "Nombre de la obra"}
```

?

Reply:
What do you mean? 
You was talking about change the "Nombre de la obra" to another better word not ?

Reply:
@Katsu08 in this case `alt` is not needed

Reply:
Oh sorry!. Thanks for the correction. I have another suggestion hehe, what do you think about to use:

```suggestion
        {data.title || "Item"}
```

Reply:
I think isn't necessary to remove the conditional! Cuz this is a good idea when the API doesn't work!

Reply:
> I think isn't necessary to remove the conditional! Cuz this is a good idea when the API doesn't work!

Hello! It's not about removing conditional, it's a good idea. I propose changing the ternary conditional for a short-circuit conditional.

I left you this documentation about that and let me know what you think if it would be good to change it.

https://medium.com/@amaliesmidth/javascript-short-circuit-conditionals-6606bdeaa30d#74e5

Reply:
It's not needed, but you can read the documentation to know about other ways to create conditions. I will merge. Good work @Allislove !